src/Platform/Bridge/Anthropic/Contract/AnthropicSet.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Platform\Bridge\Anthropic\Contract;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * @author Denis Zunke <denis.zunke@gmail.com>
+ */
+final readonly class AnthropicSet
+{
+    /** @return array<NormalizerInterface> */
+    public static function get(): array
+    {
+        return [
+            new AssistantMessageNormalizer(),
+            new DocumentNormalizer(),
+            new DocumentUrlNormalizer(),
+            new ImageNormalizer(),
+            new ImageUrlNormalizer(),
+            new MessageBagNormalizer(),
+            new ToolCallMessageNormalizer(),
+            new ToolNormalizer(),
+        ];
+    }
+}
```

Comment:
how about just 

```php
final class AnthropicContract extends Contract
{
    public static function create(): self
    {
        return parent::create([
            new AssistantMessageNormalizer(),
            new DocumentNormalizer(),
            new DocumentUrlNormalizer(),
            new ImageNormalizer(),
            new ImageUrlNormalizer(),
            new MessageBagNormalizer(),
            new ToolCallMessageNormalizer(),
            new ToolNormalizer(),
        ]);
    }
}
```

would need some changes with the parent, but would ease the handling while instantiating the Platform:
```php
$contract ?? AnthropicContract::create()
```

Reply:
Thanks! This feels like a good improvement alternative to the rector inspired sets 😊